### PR TITLE
made name of cancel button configurable, added defaults for both buttons

### PR DESCRIPTION
**Pull request by jans510**

### CODE REVIEW

File: test/modal.spec.js [hunk 1]
```diff
@@ -66,7 +67,7 @@ describe('ModalMessage', function() {
         setup(['primaryButtonClicked', 'secondaryButtonClicked']);
         let buttons = document.getElementsByTagName('button');
         expect(buttons.length).toBe(2);
-        expect(buttons[1]).innerHTML = 'Cancel';
+        expect(buttons[1]).innerHTML = 'Not OK';
```
Comment (fatroom) [inline test/modal.spec.js]: Maybe worth another test to verify that default value also set properly?

Comment (jans510) [reply test/modal.spec.js]: I will add it.

**Comment by fatroom** [comment]
👍

**Comment by jans510** [comment]
:+1:

**Comment by cberg-zalando** [comment]
:+1:

**Comment by jans510** [comment]
:+1:

**Comment by cberg-zalando** [comment]
:+1:

**Pull request merged**